docs/reference/hooks.rst

```diff
@@ -364,6 +364,34 @@ Hooks for customising the way users are directed through the process of creating
         return HttpResponse("Congrats on making content!", content_type="text/plain")
 
 
+.. _before_create_page:
+
+``before_create_page``
+~~~~~~~~~~~~~~~~~~~~~
+
+  Ran at the beginning of the "create page" view passing in the request, the parent page and page model class.
```

Comment:
Grammatically this should be "Run at the beginning...". But "Called at the beginning..." is probably clearer (and matches what we've used in other descriptions).